android/fastlane/metadata/android/en-GB/changelogs/10032.txt

```diff
@@ -0,0 +1,13 @@
+Added
+    - Basic search page for the app, accessible from the home page
```

Comment:
How about "App search functionality, accessible from the home page"? I wouldn't call it basic in the changelog.